frontend/src/components/App/Settings/SettingsCluster.tsx

```diff
@@ -168,6 +169,13 @@ export default function SettingsCluster() {
             : t('settings|Cluster Settings')
         }
         backLink
+        headerProps={{
+          actions: [
+            <Link href={createRouteURL('settings')} align="right" style={{ color: 'black' }}>
```

Comment:
Colors should not be hard coded. Instead colors should be taken from the themes.